packages/compass-e2e-tests/tests/logging.test.js

```diff
@@ -385,6 +373,11 @@ describe('Logging and Telemetry integration', function () {
           }
         });
       });
+
+      it('does not contain warnings about missing optional dependencies', function () {
```

Comment:
🎉🎉🎉

Reply:
This is the one that’s still failing because of the externalization in mongosh – don’t get excited too early :smile:

Reply:
Seems like it all green now *gets excited* 😃